scripts/exgdas_enkf_earc.py

```diff
@@ -34,9 +34,9 @@ def main():
     for key in keys:
         archive_dict[key] = archive.task_config[key]
 
-    # Also import all COM* directory and template variables
+    # Also import all COMIN* directory and template variables
     for key in archive.task_config.keys():
-        if key.startswith("COM"):
+        if key.startswith("COMIN"):
```

Comment:
Ref. the failure in https://github.com/NOAA-EMC/global-workflow/pull/2667#issuecomment-2177032111.
Following the hotfix in #2697, those `COM_ATMOS_<>_TMPL` variables are also needed to resolve the jinja2 templates (in addition to the `COMIN`)
We need another hotfix for this.

Reply:
Do you want to open an issue or just for me to submit a hotfix?

Reply:
Fixed it as part of #2667